src/core/localization/translations/en/common.json

```diff
@@ -358,11 +358,11 @@
       "freehand_geometry": "Freehand geometry",
       "to_replace_reference_area": "To replace reference area, select an area on the map and click \"Save as reference area\" on toolbar.",
       "select_are_on_the_map": "Select an area on the map and click \"Save as reference area\" on toolbar to use it in further analysis.",
-      "set_the_reference_area": "Set the reference area"
+      "set_the_reference_area": "Set reference area"
     }
   },
   "current_event": {
-    "not_found_request": "Sorry, requested event not found."
+    "not_found_request": "Sorry, requested disaster is not found."
```

Comment:
Are we sure "is not found" works here? Present tense feels weird for some reason

Reply:
Here are a few improved options for the message:

"Sorry, the requested disaster could not be found."
"Apologies, the requested disaster information is unavailable."
"Sorry, we couldn't locate the requested disaster."
"We apologize, but the disaster you requested is not found."
"The requested disaster information is not available. Please try again."